### ISSUE TITLE
Handling Unit Editor Materialentnahme Bug

### ISSUE THREAD

**Issue by metas-mk** [metasfresh/metasfresh-webui-api#561]
### Is this a bug or feature request?
Bug
### What is the current behavior?
Currently, the workflow using the "Materialentnahme" in WebUI Handling Unit Editor Window is not usable.
#### Which are the steps to reproduce?
Describing case in w101.
a) Open the Handling Unit Editor in WebUI. 
b) Press Filter Barcode. Type Barcode 1000202.
c) Select top level HU 1000202. Press action Materialentnahme.
<-- Nothing happens

What should happen?
1. The HU should disappear from the view because this happened in Swing too. This could be solved by adding a SQL Where in the Tab which does not show HU in the "Materialentnahme Locator".
<-- This does not work atm because the selected view is still shown in the initial warehouse. Only after refreshing the browser it is shown in the Materialentnahme Warehouse.
<-- This also does not work atm, because the Main view does not care about the SQL Where. The moved HU is still shown there. If you open the detail view the user gets a 404.

### What is the expected or desired behavior?
Shall work.
These are the first initial steps that are needed urgently. Next steps will follow (has to be verified).

**Comment by teosarca** [comment]
note to IT: to be tested together with https://github.com/metasfresh/metasfresh/issues/2145

**Comment by metas-ts** [comment]
also cherry-picked to `2017-09`

**Comment by metas-dh** [comment]
Results of IT1
tested in mf15

* HU disappears after action Materialentnahme: OK
* HU is displayed again after page is refreshed, Locator is changed to Materialentnahmelager: OK
* Materialentnahme not possible for HU status Shipped or Destroyed: OK
* Materialentnahme possible for HU with status Issued: NOK
* Materialentnahme possible for HU with status Planning: not sure this is OK, afaiu HUs with that status are not really existing yet?

**Comment by teosarca** [comment]
update:
* materialentnahme processes are available only for active HUs (HUStatus=A)
* @metas-mk  @metas-dh  i think we have the same problem on "move to quality warehouse", "move to another WH" actions. wdyt? if that's the case, pls create a follow-up task about that.

**Comment by metas-rc** [comment]
Results of IT

* OK: Materialennahme works only for the Active HUs.
* OK: Materialennahme cannot be started for Issued HUs.
